### PR TITLE
change getSeekRangeEnd to allow scrubbing to end of in-progress VOD

### PR DESCRIPTION
**Pull request by baconz**
Addresses #425

### CODE REVIEW

**Comment by baconz** [comment]
I signed it!

File: test/media/presentation_timeline_unit.js [hunk 1]
```diff
@@ -258,39 +260,73 @@ describe('PresentationTimeline', function() {
     });
   });
 
+  describe('getSeekRangeEnd', function() {
+    it('applies uses duration for vod', function() {
```
Comment (tdrews) [inline test/media/presentation_timeline_unit.js]: nit: should this just be "uses duration for vod"?

Comment (baconz) [inline test/media/presentation_timeline_unit.js]: Fixed. Sorry about that.

**Comment by tdrews** [comment]
lgtm

**Pull request merged**